tensorboard/compat/proto/update.sh

```diff
@@ -37,6 +37,7 @@ find tensorboard/compat/proto/ -type f  -name '*.proto' -exec perl -pi \
   -e 's|package tensorflow.tfprof;|package tensorboard;|g;' \
   -e 's|package tensorflow;|package tensorboard;|g;' \
   -e 's|tensorflow\.DataType|tensorboard.DataType|g;' \
+  -e 's|tensorflow\.TensorProto|tensorboard.TensorProto|g;' \
```

Comment:
This PR should now build!  The TF side added a new proto name that our `update.sh` script didn't replace, so this line is added.